src/gui/MainWindow.cc

```diff
@@ -805,6 +805,8 @@ MainWindow::MainWindow(const QStringList& filenames) :
   updateExportActions();
 
   activeEditor->setFocus();
+  GlobalPreferences::inst()->addColorSchemes(activeEditor->colorSchemes());   // needs to be done only once, however handled
```

Comment:
Not sure I understand this. It's now done per MainWindow, instead of only once when Preferences is initialized?

Reply:
Right, I forgot that in openscad there can be multiple MainWindow, I will add a check to insure the setting is done only once.